internal/file/watch/watch.go

```diff
@@ -133,19 +132,14 @@ func (w *watch) Start(ctx context.Context) (<-chan error, error) {
 					}
 					switch {
 					case event.Op&fsnotify.Write == fsnotify.Write && w.onWrite != nil:
-						log.Debug("Modified file: ", event.Name)
```

Comment:
maybe it is better to have a debug log like 
`Trigger onWrite hook, Modified file: %s`

Reply:
I deleted these lines because they print too many logs.
But i think it's okay to have debug logs here.
I'll restore these lines again.